src/events.ts

```diff
@@ -68,63 +63,27 @@ export const handleDeletedEvent = async event => {
     INSERT IGNORE INTO events SET ?;
   `;
   return db.queryAsync(query, [event.id, event]);
-};
-
-export async function sendEvent(event, to, method = 'POST') {
-  event.token = sha256(`${to}${serviceEventsSalt}`);
-  event.secret = sha256(`${to}${serviceEventsSalt}`);
-  const headerSecret = sha256(`${to}${process.env.SERVICE_EVENTS_SALT}`);
-  const url = to.replace('[PROPOSAL-ID]', event.id.split('/')[1]);
-  const end = timeOutgoingRequest.startTimer({ method });
-  let res;
-
-  try {
-    res = await fetch(url, {
-      method,
-      headers: {
-        'Content-Type': 'application/json',
-        Authentication: headerSecret
-      },
-      body: JSON.stringify(event),
-      timeout: HTTP_WEBHOOK_TIMEOUT
-    });
-    return res.text();
-  } catch (error: any) {
-    if (error.message.includes('network timeout')) {
-      console.error('[events] Timed out while sending the webhook', url);
-    } else {
-      console.error('[events] Error sending event data to webhook', url, JSON.stringify(error));
-    }
-    throw error;
-  } finally {
-    end({ status: res?.statusCode || 0 });
-  }
 }
 
-const sendEventToWebhookSubscribers = (event, subscribers) => {
-  Promise.allSettled(
-    subscribers
-      .filter(subscriber => [event.space, '*'].includes(subscriber.space))
-      .map(subscriber => sendEvent(event, subscriber.url, subscriber.method))
-  )
-    .then(() => console.log('[events] Process event done'))
-    .catch(e => capture(e));
-};
-
-async function processEvents(subscribers) {
-  const ts = parseInt((Date.now() / 1e3).toFixed()) - delay;
-  const events = await db.queryAsync('SELECT * FROM events WHERE expire <= ?', [ts]);
+async function processEvents() {
+  const ts = ~~(Date.now() / 1e3) - DELAY;
 
+  const events = await db.queryAsync('SELECT * FROM events WHERE expire <= ?', [ts]);
   console.log('[events] Process event start', ts, events.length);
 
   for (const event of events) {
     const proposalId = event.id.replace('proposal/', '');
-
-    // Send event to discord subscribers and webhook subscribers and then delete event from db
-    // TODO: handle errors and retry
-    if (servicePushNotifications && event.event === 'proposal/start') sendPushNotification(event);
-    sendEventToDiscordSubscribers(event.event, proposalId);
-    sendEventToWebhookSubscribers(event, subscribers);
+    const proposal = await getProposal(proposalId);
+    const subscribers = await getSubscribers(event.space);
+
+    if (proposal) {
+      // TODO: handle errors and retry
+      sendBeams(event, proposal, subscribers);
+      sendDiscord(event, proposal, subscribers);
+      sendWebhook(event, proposal, subscribers);
```

Comment:
Could create a new file providers/index.ts, with content : 
```
import { send as sendDiscord } from './discord';
import { send as sendBeams } from './beams';
import { send as sendWebhook } from './webhook';

export default function send(event, proposal, subscribers) {
  sendBeams(event, proposal, subscribers);
  sendDiscord(event, proposal, subscribers);
  sendWebhook(event, proposal, subscribers);
}
```

So that events.ts could just import 
```
import send from './providers';
```

and use a single send call

```
send(event, proposal, subscribers);
```